test/src/imgur/getTags.test.ts

```diff
@@ -1,8 +1,13 @@
+import { ImgurClient } from 'imgur'
+
 const imgurGetTagsMethod = 'biketag.images.getTags'
+
 describe(imgurGetTagsMethod, () => {
   test.todo(`${imgurGetTagsMethod} method requires ImgurHash from payload`)
```

Comment:
`getTags` has three branches to a conditional. Two require hash, but one doesn't. The conditions are:

```ts
  if (payload.tagnumbers?.length && payload.hash) {
    // ...
  } else if (payload.slugs?.length) {
    // ...
  } else if (payload.hash) {
    // ...
  }
```

💡  Suggestion: Drop this test and split next into 3 (see comments on next test)